templates/login.html

```diff
@@ -2,25 +2,26 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <title>Microsoft Identity Python Web App: Login</title>
 </head>
 <body>
     <h1>Microsoft Identity Python Web App</h1>
 
+    <ul>
     {% if user_code %}
-    <ol>
     <li>To sign in, type <b>{{ user_code }}</b> into
       <a href='{{ auth_uri }}' target=_blank>{{ auth_uri }}</a>
       to authenticate.
     </li>
     <li>And then <a href="{{ url_for('auth_response') }}">proceed</a>.</li>
-    </ol>
     {% else %}
     <li><a href='{{ auth_uri }}'>Sign In</a></li>
     {% endif %}
 
     {% if config.get("B2C_RESET_PASSWORD_AUTHORITY") %}
     <li><a href="{{config.get('B2C_RESET_PASSWORD_AUTHORITY')}}?client_id={{config.get('CLIENT_ID')}}">Reset Password</a></li>
     {% endif %}
```

Comment:
Nah. The code path ABOVE needs to be an ordered list. (If you haven't already, you can try it by comment out the "redirect_uri" line inside app.py's login(). It is fun. :-) )

But I see your point. The proper way to do it, is to revert that `<ol>` change, and then add an `<ul>` inside the `{%else%}` branch below. Also we have to remove the `<li>` tag around the B2C reset password link in line 22.

```suggestion
    {% else %}
    <ul><li><a href='{{ auth_uri }}'>Sign In</a></li></ul>
    {% endif %}

    {% if config.get("B2C_RESET_PASSWORD_AUTHORITY") %}
    <a href="{{config.get('B2C_RESET_PASSWORD_AUTHORITY')}}?client_id={{config.get('CLIENT_ID')}}">Reset Password</a>
    {% endif %}
```